src/mmw/apps/modeling/calcs.py

```diff
@@ -59,8 +61,13 @@ def point_source_pollution(geojson):
 
         if cursor.rowcount != 0:
             columns = [col[0] for col in cursor.description]
-            point_source_results = [dict(zip(columns, row)) for row in
-                                    cursor.fetchall()]
+            point_source_results = []
+            for row in cursor.fetchall():
+                point_source_row = dict(zip(columns, row))
+                for k, v in point_source_row.iteritems():
+                    if isinstance(v, decimal.Decimal):
```

Comment:
Since we _know_ which fields could be decimals, this check is unnecessary. This may be a matter of taste, but I would prefer the more Pythonic:

```diff
diff --git a/src/mmw/apps/modeling/calcs.py b/src/mmw/apps/modeling/calcs.py
index 6a21eb4..1799292 100644
--- a/src/mmw/apps/modeling/calcs.py
+++ b/src/mmw/apps/modeling/calcs.py
@@ -61,13 +61,14 @@ def point_source_pollution(geojson):
 
         if cursor.rowcount != 0:
             columns = [col[0] for col in cursor.description]
-            point_source_results = []
-            for row in cursor.fetchall():
-                point_source_row = dict(zip(columns, row))
-                for k, v in point_source_row.iteritems():
-                    if isinstance(v, decimal.Decimal):
-                        point_source_row[k] = float(v)
-                point_source_results.append(point_source_row)
+            point_source_results = [
+                dict(zip(columns,
+                         [row[0], row[1],
+                          float(row[2]) if row[2] else None,
+                          float(row[3]) if row[3] else None,
+                          float(row[4]) if row[4] else None]))
+                for row in cursor.fetchall()
+            ]
         else:
             point_source_results = []
 
```

We need those three `if` checks in there since `float(None)` raises an error.